appdev/guides/user-metrics.rst

```diff
@@ -0,0 +1,94 @@
+User metrics
+============
+
+What are user metrics?
+----------------------
+
+If you look on the lock screen, you will see a circle. Inside the circle is text. Look closer, and you’ll notice that the text contains data regarding the user’s activity. Double tap on the middle of the circle, and you will see more “metrics” about the user.
+
+.. figure:: /_static/images/appdev/guides/usermetricsimages/met1.png
+
+    This shows “7 text messages sent today.” How did it know?
+
+.. figure:: /_static/images/appdev/guides/usermetricsimages/met2.png
+
+    This is from a 3rd-party application (`nCounter <https://gitlab.com/joboticon/ncounter/>`_) that makes use of the User Metrics feature.
+
+For the most part, these messages are quite clearly state what they are counting, and which app is related. But where do these metrics come from?
+
+How can I use user metrics in my application?
+---------------------------------------------
+
+All of the following information will be based on the code for `nCounter`_
+
+First, you will need to import the module in the QML file that will handle the User Metrics:
+
+.. code:: qml
+
+        import UserMetrics 0.1
+
+(There may be updated versions of this above 0.1)
+
+Next, the specific Metric must be defined in the code as an object:
+
+.. code:: qml
+
+            Metric { // Define the Metric object.
+                property string circleMetric // Create a string-type variable called “circleMetric”. This is so you can update it later from somewhere else.
+                id: metric // Give the metric a name/id. Here it’s called “metric”
```

Comment:
This explanation of "id" seems like too much information for a seasoned QML developer, not enough for a new one... In short, a qml object's ID is how you reference it elsewhere in the code.